robyn/env_populator.py

```diff
@@ -0,0 +1,33 @@
+import os 
+from definitions import CONFIG_PATH
+
+# parse the configuration file returning a list of tuples (key, value) containing the environment variables
+def parser(config_path=CONFIG_PATH):
+    """Parse the configuration file"""
+    with open(config_path, 'r') as f:
+        for line in f:
+            if line.startswith('#'):
+                continue
+            yield line.strip().split('=')  
+
+
+# check for the environment variables set in cli and if not set them
+def load_vars(variables = None):
+    """Main function"""
+    
+    variables = parser() 
+
+    if variables is None:
+        return
+
+    for var in variables:
+        if var[0] in os.environ:
+            print("Variable {} already set".format(var[0]))
```

Comment:
We can use `logger` module here. Similar to (https://github.com/sansyrox/robyn/blob/main/robyn/__init__.py#L21) .

We will be able to control the logs using the `log-level`.

Second, instead of `format`, we can use `f-strings` as we support `Python >=3.7`. https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings

Reply:
should logs be in debug level or info level? and I changed the syntax such that f-strings are used instead

Reply:
@Shending-Help , these logs should be `debug` level in my opinion.

Reply:
Or info works too. There is a very thin line and doesn't really matter that much.